spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -239,7 +239,7 @@ It's important to remember that users must not call these methods directly.
 They are invoked by the Reactive Messaging implementation following the Reactive Streams protocol.
 
 Also the method must be implemented in a non-blocking fashion.
-For blocking transformations, asynchronous variants can be used.
+For blocking transformations, synchronous variants can be used.
```

Comment:
I believe the meaning is that you should execute blocking operations asynchronously, eg. don't ever block the reactive methods. Changing it to `synchronous` makes the sentence untrue.